syntax = "proto3";
package micro_app;




import "model/app_store/release_package.proto";

/*
GetRelatedPackages请求
*/
message GetRelatedPackagesRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页大小
    */
    int32 page_size = 2 ;
    /*
    小产品id
    */
    string app_id = 3 ;
}

/*
GetRelatedPackages返回
*/
message GetRelatedPackagesResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated app_store.ReleasePackage list = 4 ;
}

/*
GetRelatedPackagesApi返回
*/
message GetRelatedPackagesResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    micro_app.GetRelatedPackagesResponse data = 4 ;
}

